inputgraph/loader.go

```diff
@@ -454,6 +458,64 @@ func (l *loader) handleArg(ctx context.Context, cmd spec.Command, isBase bool) e
 	return nil
 }
 
+func (l *loader) handleLet(ctx context.Context, cmd spec.Command) error {
+	var opts commandflag.LetOpts
+	argsCpy := flagutil.GetArgsCopy(cmd)
+	args, err := flagutil.ParseArgsCleaned("LET", &opts, argsCpy)
+	if err != nil {
+		return errors.Wrap(err, "failed to parse LET args")
+	}
+	if len(args) != 3 {
```

Comment:
should we also check for a `=` somewhere? currently we'll accept any of `LET foo <- bar`, `LET foo gets bar`, or `LET foo = bar`
```suggestion
	if len(args) != 3 || args[1] != '=' {
```

Reply:
The reason for only checking length is that I didn't want to potentially get an index out of bound error. 
I think the rest of the validation should still occur later on when LET/SET are handled by the interperter.
Thoughts?

Reply:
added a comment instead